src/js/button.js

```diff
@@ -3,10 +3,7 @@
  */
 import ClickableComponent from './clickable-component.js';
 import Component from './component';
-import * as Events from './utils/events.js';
```

Comment:
doesn't `import './utils/events.js'` put the exports into local scope? This is an issue for a "bag of functions" module.
`import * Events from './utils/events.js'` is more explicit in what we want.